man/org.freedesktop.systemd1.xml

```diff
@@ -1486,6 +1490,11 @@ node /org/freedesktop/systemd1 {
       boot loader or initrd implementation. In these cases the respective pairs of timestamps are both 0,
       indicating that no data is available.</para>
 
+      <para><varname>UnitsReloadStartTimestamp</varname> encode <constant>CLOCK_REALTIME</constant>
+      and <constant>CLOCK_MONOTONIC</constant> microseconds timestamps (as described above). The timestamps
+      are taken every time when manager reload unit files. It could be used to notify user if manual systemd
```

Comment:
`when the manager reloads`
`notify users that daemon-reload is needed`

Reply:
Thank you.